### ISSUE TITLE
AttributeError: module 'torch' has no attribute 'flip'

### ISSUE THREAD

**Issue by ravikirancap** [mravanelli/SincNet#30]
Hi I am using torch 0.4.0 as mentioned in the README file. I get the following error. Is this because of a version problem or do I need to install additional dependancies(apart from the ones mentioned in the README)

Traceback (most recent call last):
  File "speaker_id.py", line 228, in <module>
    pout=DNN2_net(DNN1_net(CNN_net(inp)))
  File "/home/paperspace/anaconda3/envs/sincnet/lib/python3.6/site-packages/torch/nn/modules/module.py", line 491, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/paperspace/SincNet/dnn_models.py", line 448, in forward
    x = self.drop[i](self.act[i](self.ln[i](F.max_pool1d(torch.abs(self.conv[i](x)), self.cnn_max_pool_len[i]))))
  File "/home/paperspace/anaconda3/envs/sincnet/lib/python3.6/site-packages/torch/nn/modules/module.py", line 491, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/paperspace/SincNet/dnn_models.py", line 144, in forward
    band_pass_right= torch.flip(band_pass_left,dims=[1])
AttributeError: module 'torch' has no attribute 'flip'

**Comment by ravikirancap** [comment]
Tried with pytorch 1.0.1 and it is working fine

**Comment by mravanelli** [comment]
Ok..thank you! I should change the documentation..

**Comment by ravikirancap** [comment]
Another quick question (let me know if I should post as a different issue instead)

I am currently training a model (been a few hours into training already) and I noticed that the num epochs are too high. Is there a way to just freeze the current state of the model? I have used some your pytorch-kaldi toolkit and it provides checkpointing and restarting from the closest checkpoint. I want to know if this codebase also has that feature
